msdk-datamodel/src/main/java/io/github/msdk/datamodel/featuretables/ColumnName.java

```diff
@@ -28,7 +26,7 @@
     GROUPID("Group ID", Integer.class),
     MZ("m/z", Double.class),
     PPM("ppm", Double.class),
-    RT("RT", ChromatographyInfo.class),
+    RT("RT", Double.class),
     RTSTART("RT Start", Double.class),
```

Comment:
Thanks for spotting this, I will check where I need to fix this. There was an issue with one of the tests, when it was ChromatographyInfo. Maybe on of the builders was not setting the type correctly. Will fix this with the next pull request.